src/test/java/DockerTests.java

```diff
@@ -0,0 +1,92 @@
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.http.HttpMethod;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+@Testcontainers
+public class DockerTests
+{
+    private static final Logger LOG = LoggerFactory.getLogger(DockerTests.class);
+    private static final Pattern PATTERN = Pattern.compile("^[0-9]+\\.[0-9]*-.*");
+    private static final String USER_DIR = (System.getProperty("basedir") != null) ? System.getProperty("basedir") : System.getProperty("user.dir");
```

Comment:
I edited this line so it can be run without the maven property set, like when it is run locally in Intellij.

Reply:
ah yes my bad I missed to remove it. `basedir` is not needed anymore. I simplified that